docs/Maths.md

```diff
@@ -226,6 +226,108 @@ Result:
 ![](plotly_svg.png)
 
 
+## Using Pygal ##
+[Pygal](https://www.pygal.org/en/stable/) is a graph plotting library using Python. You can install Pygal using `pip install pygal` command.  
+
+`fpdf2` is able to embed the graph and charts that are generated using `Pygal` library. The following ways explain how to embed `Pygal` charts into `fpdf2` library. However, we can not embed graphs as SVG directly. Since, `Pygal` introduces <style> & <script> elements to the `SVG` images it produces ([Ref](https://github.com/Kozea/pygal/blob/3.0.0/pygal/svg.py#L449)) which currently not supported by `fpdf`. You can see the unsupported SVG features of `fpdf`[here](https://pyfpdf.github.io/fpdf2/SVG.html#currently-unsupported-notable-svg-features).
```

Comment:
```suggestion
`fpdf2` is able to embed the graph and charts that are generated using `Pygal` library. The following ways explain how to embed `Pygal` charts into `fpdf2` library. However, we can not embed graphs as SVG directly. Since, `Pygal` introduces `<style>` & `<script>` elements to the `SVG` images it produces ([Ref](https://github.com/Kozea/pygal/blob/3.0.0/pygal/svg.py#L449)) which is currently not supported by `fpdf2`. The full list of unsupported SVG features of `fpdf2` is [there](https://pyfpdf.github.io/fpdf2/SVG.html#currently-unsupported-notable-svg-features).
```
I think `<style>` & `<script>` should be escaped in backquotes, otherwise the Markdown interpreter may treat them as actual valid HTMl tags! 😊